src/pages/workspace/accounting/qbo/QuickbooksChartOfAccountsPage.tsx

```diff
@@ -0,0 +1,72 @@
+import {View} from 'react-native';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import Switch from '@components/Switch';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import FeatureEnabledAccessOrNotFoundWrapper from '@pages/workspace/FeatureEnabledAccessOrNotFoundWrapper';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import variables from '@styles/variables';
+import * as Policy from '@userActions/Policy';
+import CONST from '@src/CONST';
+
+function QuickbooksChartOfAccountsPage({policy}: WithPolicyProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const policyID = policy?.id ?? '';
+    const {enableNewCategories, pendingFields} = policy?.connections?.quickbooksOnline?.config ?? {};
+    const isSwitchOn = Boolean(enableNewCategories && enableNewCategories !== CONST.INTEGRATION_ENTITY_MAP_TYPES.NONE);
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={policyID}>
+            <FeatureEnabledAccessOrNotFoundWrapper
+                policyID={policyID}
+                featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            >
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    shouldEnableMaxHeight
+                    testID={QuickbooksChartOfAccountsPage.displayName}
+                >
+                    <HeaderWithBackButton title={translate('workspace.qbo.accounts')} />
+                    <ScrollView contentContainerStyle={[styles.pb2, styles.ph5]}>
+                        <Text style={styles.pb5}>{translate('workspace.qbo.accountsDescription')}</Text>
+                        <View style={[styles.flexRow, styles.mb2, styles.alignItemsCenter, styles.justifyContentBetween]}>
+                            <View style={styles.flex1}>
+                                <Text fontSize={variables.fontSizeNormal}>{translate('workspace.qbo.accountsSwitchTitle')}</Text>
+                            </View>
+                            <OfflineWithFeedback pendingAction={pendingFields?.enableNewCategories}>
+                                <View style={[styles.flex1, styles.alignItemsEnd, styles.pl3]}>
+                                    <Switch
+                                        accessibilityLabel={translate('workspace.qbo.accounts')}
+                                        isOn={isSwitchOn}
+                                        onToggle={() =>
+                                            Policy.updatePolicyConnectionConfig(
```

Comment:
ah I didn't know you actually wrote the code to call the API

anyway, these screens are behind the beta, so I think we're okay doing so